### PR TITLE
testing: assert that install.sh works correctly

### PR DESCRIPTION
**Pull request by dweomer**
For every change that modifies the installer script, run some quick assertions against:
- CentOS 7
- CentOS 8
- MicroOS

Thanks, Github Actions and Vagrant!

Signed-off-by: Jacob Blain Christen <jacob@rancher.com>

### CODE REVIEW

**Comment by dweomer** [comment]
[Will It Blend](https://www.youtube.com/willitblend)

**Comment by dweomer** [comment]
Flakes are detrimental to my chill.

File: tests/install/centos7/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+RKE2 Install on CentOS 8
```
Comment (dereknola) [inline tests/install/centos7/README.md:1]: All of theses READMEs are the same except for the OS that is being tested, can you make 1 document either under the `tests/install` folder or add a section to https://github.com/rancher/rke2/blob/master/developer-docs/testing.md

**Review by dereknola — CHANGES_REQUESTED** [review]

File: tests/install/centos7/README.md (hunk 1 above)
Comment (dweomer) [reply tests/install/centos7/README.md:1]: You'll notice that the MicroOS has additional details. I expect these to diverge per OS over time and chose to not assume that someone looking to run some quick tests will know how to navigate our documentation.

**Review by dereknola — APPROVED** [review]

**Pull request merged**